### ISSUE TITLE
Duplicate of #840.

### ISSUE THREAD

**Issue by Source61** [edubart/otclient#1146]
Duplicate of #840.

_Originally posted by @diath in https://github.com/edubart/otclient/issues/1109#issuecomment-781453720_

Just thought I'd add that this is an invalid duplicate.
I'm not clicking multiple times in the recording, only once, and the client isn't freezing or crashing either, player is just running in an endless circle.
Since I know the solution to issue #1109 I know that it isn't related to any of the issues brought up in issue #840 such as heavy use of A*.
